### PR TITLE
Update/across session creds

### PR DESCRIPTION
**Pull request by gabeabrams**
This is part of a large across-caccl change to switch access tokens from being in the session to being in the tokenStore. This will solve token race conditions between sessions of the same user and will make it possible for users to be logged into the same app on different machines without causing issues with Canvas, though this does make it possible for users to get closer to throttling limits.

### CODE REVIEW

File: client/cached.js [hunk 1]
```diff
@@ -2,6 +2,14 @@ const initCACCL = require('.');
...
+/**
+ * Memoized version of the initialization function. If given the same config
+ *   options, this function will return the previously initialized caccl
+ *   instance
+ * @author Gabe Abrams
+ * @param {object} config - the same arguments as allowed in /client/index.js
+ * @return {object} same response as /client/index.js
+ */
 module.exports = (config = {}) => {
   const cacheKey = JSON.stringify(config);
```
Comment (lbjay) [inline client/cached.js]: Not related to these changes but it seems possible for this to lead to cache misses if the json doesn't alwasy serialize in the same order. Searching npmjs shows a few "deterministic" stringify packages: https://www.npmjs.com/search?q=stable%20stringify. Just a thought.

File: server/index.js [hunk 2]
```diff
@@ -171,7 +170,7 @@ module.exports = (oldConfig = {}) => {
         ? config.canvasHost
         : req.session.canvasHost || config.canvasHost
     );
-    const accessToken = (req.session.accessToken || config.accessToken);
+    const accessToken = (req.accessToken || config.accessToken);
```
Comment (lbjay) [inline server/index.js:173]: I'm assuming this will make sense when i get to some of the other PRs

**Review by nataliemona — APPROVED** [review]

File: client/cached.js (hunk 1 above)
Comment (gabeabrams) [reply client/cached.js]: Great point! Changed!

File: server/index.js (hunk 2 above)
Comment (gabeabrams) [reply server/index.js:173]: I hope so!

**Pull request merged**